shim.js

```diff
@@ -5535,6 +5535,10 @@ require.define("/shim.coffee", function (require, module, exports, __dirname, __
         if (cb == null) cb = function() {};
         return cb(page.injectJs(js));
       },
+      clearMemoryCache: function(cb) {
```

Comment:
OK, sorry was working through something quickly and missed that this was in coffeescript. I'll do it later.